src/legacy/core_plugins/kibana/public/management/sections/settings/components/advanced_settings_voice_announcement/advanced_settings_voice_announcement.tsx

```diff
@@ -40,9 +40,15 @@
 import React, { Component } from 'react';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { EuiScreenReaderOnly, EuiDelayRender } from '@elastic/eui';
+import { FieldSetting } from '../../types';
 
-export class AdvancedSettingsVoiceAnnouncement extends Component {
-  shouldComponentUpdate = nextProps => {
+interface Props {
+  queryText: string;
```

Comment:
// nit
Is this the same as `query` in `advanced_settigns.tsx` props?
If so, lets go with either `query` or `queryText`?